### PR TITLE
Story 161 memberids profiles

### PR DESCRIPTION
**Pull request by jessesnyder**
@cguardia or @alecpm: ready for review, feedback or merge.

### CODE REVIEW

File: src/isaw.policy/src/isaw/policy/profiles/default/memberdata_properties.xml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0"?>
 <object name="portal_memberdata" meta_type="PlonePAS MemberData Tool">
   <property name="CVReference" type="string"></property>
+  <property name="ProfileReference" type="string"></property>
```
Comment (alecpm) [inline src/isaw.policy/src/isaw/policy/profiles/default/memberdata_properties.xml]: You should add an upgrade step to actually apply this change to the portal_memberdata tool.

File: src/isaw.facultycv/isaw/facultycv/content/actions.py [hunk 2]
```diff
@@ -1,17 +1,12 @@
...
-from collective.portlet.relateditems import relateditems
 
 
 def initial_setup(obj, event):
     # Create profile for initial CV
     obj.invokeFactory("CV", id="cv", title="Curriculum Vitae")
 
     # Set the reference of profile to the CV
-    print "Setting reference to CV" 
+    print "Setting reference to CV"
```
Comment (alecpm) [inline src/isaw.facultycv/isaw/facultycv/content/actions.py]: Let's get rid of this. If it's important use log()

File: src/isaw.facultycv/isaw/facultycv/content/profile.py [hunk 3]
```diff
@@ -26,96 +26,101 @@
     ),
 
     atapi.ReferenceField(
-        name = 'ProfileRef',
+        name='ProfileRef',
```
Comment (alecpm) [inline src/isaw.facultycv/isaw/facultycv/content/profile.py]: Maybe keep whitespace fixes on their own branch (or at least dedicated commits) in the future.

**Comment by jessesnyder** [comment]
Thanks @alecpm, I think we're all set now.

**Pull request merged**